src/services/api/createAppApiClient.ts

```diff
@@ -126,3 +127,20 @@ const getTasks =
       return res.data
     } catch (err) {}
   }
+export type DeleteTaskByIdRequest = {
+  _id: string
+}
+
+export type DeleteTaskByIdResponse = {
+  success: boolean
+  data: Task
+}
+
+const DeleteTaskById =
```

Comment:
```suggestion
const deleteTaskById =
```
tên hàm sao lại capitalize?